### PR TITLE
Update README quick start section

### PR DESCRIPTION
**Pull request by ArangoGutierrez**
Signed-off-by: Carlos Eduardo Arango Gutierrez <carangog@redhat.com>

#### What type of PR is this?

/kind documentation
#### Which issue(s) this PR fixes:

Fixes #

#### Special notes for your reviewer:

Document new step

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,6 +24,7 @@ The controller will run in the `kueue-system` namespace.
 Then, you can and apply some of the [samples](config/samples):
 
 ```
+kubectl apply -f kueue_v1alpha1_resourceflavor
```
Comment (alculquicondor) [inline README.md:27]:
```suggestion
kubectl apply -f config/samples/kueue_v1alpha1_resourceflavor.yaml
```

Comment (ArangoGutierrez) [reply README.md]: Ups...my honest bad, sorry 🙈

**Review by alculquicondor — COMMENTED** [review]
squash

**Pull request merged**